README.md

```diff
@@ -60,6 +60,9 @@ Emoji | Represents | Links to | Comment
 📢 | Talks | the slides/recording/repo/etc.
 🎨 | Design | the logo/iconography/visual design/etc.
 👀 | Reviewed Pull Requests
+💵 | Financial Support | relevant page
+🔍 | Funding Finders
```

Comment:
Interesting. Do you think that this could be:

```
🔍 | Funding/Grant Finders
```

Or are they distinct enough to have a separate emoji for each?

```
🔎 | Grant Finders
🔍 | Funding Finders
```

😅